src/main/kotlin/Main.kt

```diff
@@ -1,5 +1,5 @@
-package org.damascus
-
-fun main() {
-
-}
\ No newline at end of file
+//package org.damascus
+//
+//fun add(a: Int, b: Int) {
+//    return (a + b)
+//}
```

Comment:
Please don't change files outside of your branch's scope to make future reviews easier.